src/common/config_opts.h

```diff
@@ -829,6 +829,7 @@ OPTION(osd_pg_epoch_persisted_max_stale, OPT_U32, 150) // make this < map_cache_
 
 OPTION(osd_min_pg_log_entries, OPT_U32, 3000)  // number of entries to keep in the pg log when trimming it
 OPTION(osd_max_pg_log_entries, OPT_U32, 10000) // max entries, say when degraded, before we trim
+OPTION(osd_force_recovery_pg_log_entries, OPT_U32, 13000) // max entries before force recovery
```

Comment:
Perhaps we should make this option a (float) multiple of osd_max_pg_log_entries?  That way it will always be, say, 30% past whatever max is so that it doesn't also have to be configured if the min/max options have been adjusted.